packages/ng/src/app/date-range-picker/date-range-picker.models.ts

```diff
@@ -0,0 +1,11 @@
+import * as moment from 'moment';
+
+export interface DateRangeSelectChoice {
+	range: DateRange;
+	label: string;
+}
+
+export interface DateRange {
+	dateMin: moment.Moment;
+	dateMax: moment.Moment;
+}
```

Comment:
also, use `start` and `end`, you're gonna need `min` and `max` when it will support some min and max